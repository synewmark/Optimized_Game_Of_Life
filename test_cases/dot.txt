version https://git-lfs.github.com/spec/v1
oid sha256:f0e05ed5d26837bc8d5f273421de164ee0645de077e3bab929a132374603ad6f
size 33
