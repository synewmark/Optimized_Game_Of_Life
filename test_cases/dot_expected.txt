version https://git-lfs.github.com/spec/v1
oid sha256:c4f3698728239d8fc09c7c6e0a73be7311218c6438d3a4a6e4ec5b6c7d262676
size 22
